src/reconnection-service.controller.ts

```diff
@@ -37,8 +37,21 @@ export class ReconnectionServiceController {
       case 'failed':
         return this.graphUpdateQueue.getFailed();
 
+      case 'waiting':
+        return this.graphUpdateQueue.getWaiting();
+
       default:
         throw new HttpException('Unrecognized job status', HttpStatus.BAD_REQUEST);
     }
   }
+
+  @Post('queue/resume')
+  async resumeQueue() {
+    await this.graphUpdateQueue.resume();
+  }
+
+  @Post('queue/pause')
+  async pauseQueue() {
+    await this.graphUpdateQueue.pause();
+  }
```

Comment:
@wilwade for this https://github.com/AmplicaLabs/reconnection-service/issues/63

you have any suggestions around what all methods we should expose in prod?

Reply:
Here's my rule of thumbs for it: Nothing that exposes user data

Perhaps also limit the most "dangerous" endpoints, but otherwise all should be safe.

We should make sure that there is at least an "API Key" to protect them however.